components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/api/impl/BusinessRulesApiServiceImpl.java

```diff
@@ -492,6 +512,14 @@ public Response updateBusinessRule(Request request, Object businessRule,
                     responseData.add("Deployment Failure");
                     responseData.add("Failed to deploy the business rule");
                     break;
+                case (TemplateManagerConstants.UNDEPLOYMENT_FAILURE):
```

Comment:
Why we need undeployment in updating business rules?

Reply:
yes, we don't want un-deployment failure state here. I will remove it.

Reply:
undeployment removed from update business rule